neon-mmd/websurfx
# Provide two new famous with one custom colorshemes

## What does this PR do?
This PR introduces three new color schemes, namely Tokyo Night, One Dark, and Dark Chocolate. The changes in this PR are focused on adding new color schemes to the existing set of options. The algorithms, design, and architecture of the application remain unchanged. The new color schemes are defined in the CSS code, which includes the color variables and their corresponding values for each scheme.

## Why is this change important?
This change is important as it enhances the customization options for users by offering popular dark color schemes. It allows users to personalize the appearance of the application according to their preferences and create a more visually appealing and comfortable experience.

## How to test this PR locally?

To test this PR locally, you can follow these steps listed below:

1. Clone the repository or download the PR branch.
2. Navigate to the CSS file or relevant code section where the color schemes are defined.
3. Replace the existing code with the new color schemes (Tokyo Night, One Dark, Dark Chocolate) provided in the PR.
4. Save the changes and run the application.
5. Verify that the new color schemes are applied correctly and check if they meet the desired visual style and readability.
6. Test the application's functionality and ensure that the color schemes do not affect usability or accessibility negatively.
7. Provide feedback on any issues or improvements related to the new color schemes during the testing process.

## Author's checklist

- [x] add [Tokyo Night](https://imgur.com/a/ejWFatr) colorscheme
- [x] add [One Dark](https://imgur.com/a/HBEi1TO) colorscheme
- [x] add [Dark Chocolate](https://imgur.com/a/VX6A449) colorscheme


## Related issues

Closes #77
